validator/val_utils.c

```diff
@@ -775,7 +812,8 @@ val_dsset_isusable(struct ub_packed_rrset_key* ds_rrset)
 			(int)ds_get_key_algo(ds_rrset, 0));
 		if(lt) snprintf(aerr, sizeof(aerr), "%s", lt->name);
 		else snprintf(aerr, sizeof(aerr), "%d",
-			(int)ds_get_key_algo(ds_rrset, 0));
+			(int)ds_get_key_algo(ds_rrset, i));
+
```

Comment:
```suggestion
			(int)ds_get_key_algo(ds_rrset, 0));
```
Is this the result of a wrong merge? It seems to undo a commit that fixes a bug here by s/i/0/.